packages/lib-content/src/screens/Volunteer/Volunteer.js

```diff
@@ -1,15 +1,257 @@
+                  href='https://www.zooniverse.org/accounts/register'
+                />
+              ]}
+            />
+          </Paragraph>
+          <Paragraph>
+            <Trans
+              i18nKey='Volunteer.register.paragraphs.five'
+              t={t}
+              components={[
+                <Anchor
+                  key='register-spanish'
+                  href='https://www.zooniverse.org/accounts/register?language=es'
+                />,
+                <Anchor
+                  key='register-french'
+                  href='https://www.zooniverse.org/accounts/register?language=fr'
+                />
+              ]}
+            />
+          </Paragraph>
+          <StyledRegister
+            margin={{ top: '15px', bottom: 'medium' }}
+            alignSelf='center'
+            href='https://www.zooniverse.org/accounts/register'
+            label={t('Volunteer.register.button')}
+          />
+
+          {/** Beta Testing */}
+          <Heading level='3' size='1.13rem' weight='bold' color={headingColor}>
+            {t('Volunteer.beta.heading')}
+          </Heading>
+          <Paragraph>{t('Volunteer.beta.paragraphs.one')}</Paragraph>
+          <Paragraph margin='0'>
+            <Trans
+              i18nKey='Volunteer.beta.paragraphs.two'
+              t={t}
+              components={[
+                <Anchor
+                  key='lab-policies-page'
+                  href='https://help.zooniverse.org/getting-started/lab-policies'
+                />
+              ]}
+            />
+          </Paragraph>
+          <Paragraph>
+            <Trans
+              i18nKey='Volunteer.beta.paragraphs.three'
+              t={t}
+              components={[
+                <Anchor
+                  key='email-settings'
+                  href='https://www.zooniverse.org/settings/email'
+                />
+              ]}
+            />
+          </Paragraph>
+          <StyledBeta
+            margin={{ top: '15px', bottom: 'medium' }}
+            alignSelf='center'
+            href='https://www.zooniverse.org/settings/email'
+            label={t('Volunteer.beta.button')}
+          />
+
+          {/** Moderate */}
+          <Heading level='3' size='1.13rem' weight='bold' color={headingColor}>
+            {t('Volunteer.moderate.heading')}
+          </Heading>
+          <Paragraph>
+            <Trans
+              i18nKey='Volunteer.moderate.paragraphs.one'
+              t={t}
+              components={[
+                <Anchor
+                  key='what-are-moderators'
+                  href='https://docs.google.com/document/d/1L8LwYy_uUxwX1NqE5sXi0fnrjZKG1DZu1fWLath9BOE'
+                />
+              ]}
+            />
+          </Paragraph>
+          <Paragraph margin='0'>
+            <Trans
+              i18nKey='Volunteer.moderate.paragraphs.two'
+              t={t}
+              components={[<Anchor key='contact-us' href='/about#contact' />]}
```

Comment:
Note that this href can be a relative path because the About pages are also in app-root. Pages that use PFE or other Zoo-apps are written as absolute paths in this component.